lua/neogit/popups/branch/actions.lua

```diff
@@ -64,10 +68,7 @@ local function create_branch(popup, prompt, checkout)
     return
   end
 
-  local name = input.get_user_input("Create branch", {
-    strip_spaces = true,
-    default = popup.state.env.suggested_branch_name,
-  })
+  local name = get_branch_name_user_input("Create branch", popup.state.env.suggested_branch_name)
```

Comment:
I couldn't figure out how to test this behavior. How can I cause a situation where `popup.state.env.suggested_branch_name` is defined?

Reply:
I found that, but what do I need in the UI in order to get there?